### PR TITLE
remove ape

### PR DESCRIPTION
**Pull request by wakamex**

### CODE REVIEW

**Comment by wakamex** [comment]
moved to draft, my intention wasn't to remove ape, just to make the streamlined pipeline work better (just so happens it works better without ape)
keeping as draft for @dpaiton to reference as he works on a more fullsome removal of ape

File: elfpy/bots/bot_info.py [hunk 1]
```diff
@@ -16,31 +15,23 @@ class AgentConfig:
...
+    init_kwargs : dict
+        A dictionary of keyword arguments for the policy constructor
     """
 
-    # TODO: We should be able to enable this again once we remove evm_bots
+    # lots of configs!
     # pylint: disable=too-many-instance-attributes
 
     policy: Type[BasePolicy]
-    index: int | None = None  # TODO: Make this required when we remove evm_bots
```
Comment (wakamex) [inline elfpy/bots/bot_info.py:37]: ???

Comment (dpaiton) [reply elfpy/bots/bot_info.py:37]: the TODO was dumb -- we don't use it anymore

File: elfpy/bots/environment_config.py [hunk 2]
```diff
@@ -21,14 +20,12 @@ class EnvironmentConfig(types.FrozenClass):
...
-    # Logical username for who is running bots
-    username_register_url: str = "http://localhost:5001"
-    # whether to run on alchemy
-    alchemy: bool = False
-    # url for retrieving the contract artifacts
-    artifacts_url: str = "http://localhost:80"
-    # whether to run on devnet
-    devnet: bool = True
+    # hostname for container URLs
+    hostname: str = "http://localhost"
```
Comment (slundqui) [inline elfpy/bots/environment_config.py:24]: This is making the assumption that these services are all on the same machine. Is this always the case?

**Review by sentilesdal — APPROVED** [review]

File: elfpy/bots/environment_config.py [hunk 3]
```diff
@@ -21,14 +20,12 @@ class EnvironmentConfig(types.FrozenClass):
...
+    # ports for container URLs
+    artifacts_port: str = "80"
```
Comment (slundqui) [inline elfpy/bots/environment_config.py:26]: Why are these not integers?

File: examples/eth_bots/config.py [hunk 4]
```diff
@@ -58,23 +58,16 @@
...
     random_seed=1234,
-    # TODO: should just use
-    # hostname = http://localhost
-    # artifacts_port = 80
-    # rpc_port = 8545
-    # user_registry_port = 5002
-    artifacts_url="http://localhost:80",
-    rpc_url="http://localhost:8545",
-    username_register_url="http://localhost:5002",
+    hostname="http://localhost",
```
Comment (slundqui) [inline examples/eth_bots/config.py:68]: Likewise, same comment. Are these always going to live on the same server?

File: examples/eth_bots/config.py [hunk 5]
```diff
@@ -58,23 +58,16 @@
...
+    artifacts_port="80",
```
Comment (slundqui) [inline examples/eth_bots/config.py:69]: Should be int

**Review by slundqui — COMMENTED** [review]
Should also look over docker build and various CI workflows for ape. Grep ape should do the trick

**Pull request merged**